app/controllers/legal_docs_controller.rb

```diff
@@ -24,10 +24,8 @@ def create
 
   def index
     smart_listing_create :legal_docs, 
-                        LegalDoc.accessible_by(current_ability), 
-                        partial: "legal_docs/listing", 
-                        default_sort: { created_at: "desc" }
-    
+                        LegalDoc.all, 
```

Comment:
@browning25  what is the error you are getting? C/P Here.

also, check out the employees controller. it may help out a bit